framework/src/util/AnyMap.cpp

```diff
@@ -23,27 +23,13 @@
 #include "cppmicroservices/AnyMap.h"
 
 #include <stdexcept>
+#include <algorithm>
+#include <functional>
```

Comment:
Just looking at the diff, it is not clear to me why those include directives were added.

Reply:
Removed. They were there in an earlier iteration that had the implementation for any_map_cihash::operator() in the .cpp file instead of the header.